project-folder/CNN.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p><code></code><code></p><p>#Somehow it improved using the validation train and test that was randomly split</p><p>#Early stopping was not good, reduced learning rate better</p><p></code><code></code></p><p> </p><p>I don't think your early stopping did anything :P </p><p>You set <code>epochs=30</code> and the model trained for 30 epochs. Why do you think reduced learning rate is better?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>